janusgraph-cassandra/src/main/java/org/janusgraph/diskstorage/cassandra/AbstractCassandraStoreManager.java

```diff
@@ -62,73 +62,51 @@ public static Partitioner getPartitioner(String className) {
     public static final ConfigNamespace CASSANDRA_NS =
             new ConfigNamespace(GraphDatabaseConfiguration.STORAGE_NS, "cassandra", "Cassandra storage backend options");
 
-    public static final ConfigOption<String> CASSANDRA_KEYSPACE =
-            new ConfigOption<String>(CASSANDRA_NS, "keyspace",
-            "The name of JanusGraph's keyspace.  It will be created if it does not exist. " +
-                    "If it is not supplied, but graph.graphname is, then the the keyspace will be set to that.",
-            ConfigOption.Type.LOCAL, "janusgraph");
+    public static final ConfigOption<String> CASSANDRA_KEYSPACE = new ConfigOption<>(CASSANDRA_NS, "keyspace",
+        "The name of JanusGraph's keyspace.  It will be created if it does not exist. "
+            + "If it is not supplied, but graph.graphname is, then the the keyspace will be set to that.", ConfigOption.Type.LOCAL, "janusgraph");
```

Comment:
Can you check indentation and line length here and below? I'd try to match line length used in module (120?).

Reply:
ill fix the indentation. seems like IntelliJ reformatted some stuff when changing the diamond operators